### ISSUE TITLE
Model Snapshot: Implement Inheritance Support

### ISSUE THREAD

**Issue by bricelam** [aspnet/EntityFramework#2597]
Depends on #1704

**Comment by rowanmiller** [comment]
@smitpatel is this done?

**Comment by smitpatel** [comment]
This is done. ModelSnapshot includes call to BaseType in entity declaration, which should do everything required. Closing this issue. If we find something missing in future, will file a new issue.
